src/components/shared/Extensions/Accordion/hooks.ts

```diff
@@ -1,7 +1,7 @@
 import { useState } from 'react';
 
 export const useAccordion = () => {
-  const [openIndex, setOpenIndex] = useState<number | undefined>(0);
+  const [openIndex, setOpenIndex] = useState<number | undefined>(-1);
```

Comment:
why this changes?

Reply:
Is there any requirement that the accordion has to be opened by default?
I use this functionality in Transactions tabs and their default state is no one item opened.